src/utils.py

```diff
@@ -334,11 +339,17 @@ def book_appointment(request_header, details):
                 print('TOKEN INVALID')
                 return False
 
+            if resp.status_code == 409:
+                print('Slots taken')
+                return False
+
             elif resp.status_code == 200:
                 beep(WARNING_BEEP_DURATION[0], WARNING_BEEP_DURATION[1])
                 print('##############    BOOKED!  ############################    BOOKED!  ##############')
                 print("                        Hey, Hey, Hey! It's your lucky day!                       ")
                 print('\nPress any key thrice to exit program.')
+                say("Hey, Hey, Hey! It's your lucky day! , Press any key thrice to exit program!")
```

Comment:
This may not be necessary since the slot has been booked by this line and not sure letting users press the key thrice would be a value add. Lmk if you think otherwise